aries-site/src/pages/components/toast-notifications.mdx

```diff
@@ -0,0 +1,98 @@
+import { Example } from '../../layouts';
+import { ToastNotificationExample, ToastDiagram } from '../../examples';
+import { StatusGoodSmall } from 'grommet-icons/icons/StatusGoodSmall';
+import { StatusWarningSmall } from 'grommet-icons/icons/StatusWarningSmall';
+import { StatusUnknownSmall } from 'grommet-icons/icons/StatusUnknownSmall';
+import { Box } from 'grommet';
+
+<Example
+  docs="https://v2.grommet.io"
+  code={[
+      "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/toast-notifications/ToastNotificationExample.js",
+      "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/notifications/Notification.js",
+  ]}
+  figma="https://www.figma.com"
+>
+  <ToastNotificationExample />
+</Example>
+
+## Guidance
+
+Everything that you need to know about toasts and more.
+
+### When should you use a Toast Notification?
+
+Toasts should be considered when displaying these types of information to the user:
+
+- Low attention messages that do not require user action
+- Singular status updates
+- Confirmations
+- Information that does not need to be followed up
+
+Do not use Toasts if the information contains the following:
+
+- High attention and crtitical information
+- Time-sensitive information
+- Requires user action or input
+- Batch updates
+
+Due to their limited use cases, toasts are advised to be used sparingly or even substituted entirely for a different type of notification. In no way are toasts mandatory elements for a platform’s notification system.
+
+## Anatomy 
+
+<ToastDiagram />
+
+Toasts are compromised of three main elements:
+
+1. Status Indicator
+2. Content (Title + Message)
+3. Close button
+
+Aside from this, all toasts have a fixed width size at Medium (384px) while the height changes based off the amount of content. 
+
+### Status Indicator
+
+Avoid using toasts for very critical information or alerts. Users may accidentally dismiss or entirely miss a toast, so it is best to avoid this case for important information. Additionally, avoid using toasts, critical or not, if it requires a user action. 
+
+The status indicators used for toasts are strictly for OK, Warning, and Unknown statuses.
+
+<Box direction="row" gap="medium" pad={{bottom: 'medium'}}>
+    <StatusGoodSmall color="status-ok" />
+    <StatusWarningSmall color="status-warning" />
+    <StatusUnknownSmall color="status-unknown" />
+</Box>
+
+You can read more guidance about status indicators [here on this page](/components/status-indicator).
```

Comment:
Maybe rephrase so that link title is more informative
```suggestion
You can read more guidance about [status indicators](/components/status-indicator).
```